src/components/sdk/DeviceSelection/MicSelection/index.tsx

```diff
@@ -5,21 +5,28 @@ import React from 'react';
 
 import { useSelectAudioInputDevice } from '../../../../hooks/sdk/useSelectAudioInputDevice';
 import { useAudioInputs } from '../../../../providers/DevicesProvider';
+import { DeviceConfig } from '../../../../types';
 import DeviceInput from '../DeviceInput';
 
 interface Props {
   /** The message that will be shown when no microphone devices are found. */
   notFoundMsg?: string;
   /** The label that will be shown for microphone selection, it defaults to `Microphone source`. */
   label?: string;
+  /** A boolean that determines whether or not to include additional sample audio input devices */
```

Comment:
nit: End with a "dot". This comments applies for other components as well.

Reply:
Suggest adding two things:
1. What is sample devices?
2. A note that we will be deprecating this prop and remove the sample devices in next major version release.